gatherers/manifest.js

```diff
@@ -54,34 +32,62 @@ class Manifest extends Gather {
     };
   }
 
+  /**
+   * Creates a listener for manifests in network requests, a load timeout, and
+   * a promise they resolve when complete.
+   * @private
+   */
+  static _resolveOnManifest() {
```

Comment:
actually, maybe there is a nice way to build this more into the driver. I'll explore after I figure out why this sometimes doesn't work